monai/engines/evaluator.py

```diff
@@ -45,9 +45,13 @@ class Evaluator(Workflow):
         epoch_length: number of iterations for one epoch, default to `len(val_data_loader)`.
         non_blocking: if True and this copy is between CPU and GPU, the copy may occur asynchronously
             with respect to the host. For other cases, this argument has no effect.
-        prepare_batch: function to parse image and label for current iteration.
+        prepare_batch: function to parse expected data (usually `image`, `label` and other network args)
+            from `engine.state.batch` for every iteration, for more details please refer to:
+            https://github.com/pytorch/ignite/blob/v0.4.7/ignite/engine/__init__.py#L33.
```

Comment:
This one is a bit tricky to exact find the docs for, but how about this one ?
https://pytorch.org/ignite/generated/ignite.engine.create_supervised_trainer.html

Reply:
Hi @vfdev-5 ,

Sure, updated in the latest commit.

Thanks.